domenetjenester/datavarehus/src/main/java/no/nav/foreldrepenger/datavarehus/v2/StønadsstatistikkUttakPeriodeMapper.java

```diff
@@ -67,25 +71,72 @@ class StønadsstatistikkUttakPeriodeMapper {
             gradering, samtidigUttakProsent);
     }
 
-    private static StønadsstatistikkUttakPeriode.Forklaring utledForklaringAvslag(ForeldrepengerUttakPeriode periode) {
-        //TODO
-        return null;
+    private static Forklaring utledForklaringAvslag(ForeldrepengerUttakPeriode periode) {
+        if (periode.getResultatÅrsak().getUtfallType() != PeriodeResultatÅrsak.UtfallType.AVSLÅTT) {
+            throw new IllegalArgumentException("Forventer periode med avslågsårsak. Fikk " + periode.getResultatÅrsak()
+                + " for periode " + periode.getTidsperiode());
+        }
+
+        return switch (periode.getResultatÅrsak()) {
+            case AKTIVITETSKRAVET_ARBEID_I_KOMB_UTDANNING_IKKE_DOKUMENTERT,
+                AKTIVITETSKRAVET_ARBEID_IKKE_DOKUMENTERT,
+                AKTIVITETSKRAVET_ARBEID_IKKE_OPPFYLT,
+                AKTIVITETSKRAVET_INNLEGGELSE_IKKE_DOKUMENTERT,
+                AKTIVITETSKRAVET_INTROPROGRAM_IKKE_DOKUMENTERT,
+                AKTIVITETSKRAVET_KVP_IKKE_DOKUMENTERT,
+                AKTIVITETSKRAVET_SYKDOM_ELLER_SKADE_IKKE_DOKUMENTERT,
+                AKTIVITETSKRAVET_UTDANNING_IKKE_DOKUMENTERT,
+                AKTIVITETSKRAVET_MORS_DELTAKELSE_PÅ_INTRODUKSJONSPROGRAM_IKKE_OPPFYLT,
+                AKTIVITETSKRAVET_MORS_DELTAKELSE_PÅ_KVALIFISERINGSPROGRAM_IKKE_OPPFYLT,
+                AKTIVITETSKRAVET_MORS_INNLEGGELSE_IKKE_OPPFYLT,
+                AKTIVITETSKRAVET_MORS_SYKDOM_IKKE_OPPFYLT,
+                AKTIVITETSKRAVET_OFFENTLIG_GODKJENT_UTDANNING_I_KOMBINASJON_MED_ARBEID_IKKE_OPPFYLT,
+                AKTIVITETSKRAVET_OFFENTLIG_GODKJENT_UTDANNING_IKKE_OPPFYLT,
+                MORS_MOTTAK_AV_UFØRETRYGD_IKKE_OPPFYLT,
+                BARE_FAR_RETT_MOR_FYLLES_IKKE_AKTIVITETSKRAVET,
+                BARE_FAR_RETT_MANGLER_MORS_AKTIVITET -> Forklaring.AVSLAG_AKTIVITETSKRAV;
+            case SØKNADSFRIST -> Forklaring.AVSLAG_SØKNADSFRIST;
+            case HULL_MELLOM_FORELDRENES_PERIODER,
+                MOR_TAR_IKKE_ALLE_UKENE,
+                BARE_FAR_RETT_IKKE_SØKT,
```

Comment:
Se kommentar over. Vurder egen forklaring for BFHR ikke søkt - 4102 i bruk siden aug 2022.